tests/en/media_player_HassMediaNext.yaml

```diff
@@ -1,9 +1,9 @@
-language: en
+language: fr
 tests:
   - sentences:
-      - "next item on TV"
```

Comment:
You're modifying the English file here. It should be the French.

Reply:
oops, thank you, I fix that